src/pycram/robot_description.py

```diff
@@ -167,6 +173,37 @@ def get_actuator_for_joint(self, joint: str) -> Optional[str]:
             return self.joint_actuators.get(joint)
         return None
 
+
+    def get_actuated_joint_names(self, part_name) -> Optional[str]:
+        """
+        Given a part name, returns the actuated joints.
+
+        Args:
+            part_name (str): part name
+
+        Returns:
+            List of actuated joint names: or None
+        """
+
+        return self.joint_actuators.get(part_name)
+
+    def get_actuated_joint_indices(self, joint_name) -> Tuple[Optional[str],Optional[int], Optional[int]]:
+        """
+        Given a joint name, returns the indices of the part and the joint within that part.
```

Comment:
I don't get what this method does. What is a "part"?

Reply:
There is a dict of part_name , actuated joints list. 
Part can be (torso, left_arm,right_arm,head) in the icub case. a part can have multiple kinematic chains. 
for example because of the fingers or binocular vision. 

For icub, each part is controlled by a single command.

Reply:
I see, could you add that to the doc string?